src/OSPSuite.Core/Domain/Data/DataRespositoryExtensions.cs

```diff
@@ -43,6 +43,8 @@ public static DataColumn FirstDataColumn(this DataRepository dataRepository)
 
       private static string valueMapper(IEnumerable<IExtendedProperty> properties, IExtendedProperty value)
       {
+         if (value.ValueAsObject == null)
+            return "";
```

Comment:
string.empty instead of "" please

Reply:
and in fact, it might work to just use ? in `value.ValueAsObject?.ToString()`

Last, please add a test :)

Reply:
I tried yesterday with value.ValueAsObject?.ToString() but did not work. Today, after writing the test I tried again and it still throws  so I just switch to string.Empty.